squarelet/users/serializers.py

```diff
@@ -147,3 +153,12 @@ def __init__(self, *args, **kwargs):
             if self.instance and not self.instance.can_change_username:
                 # pylint: disable=invalid-sequence-index
                 self.fields["username"].read_only = True
+
+
+class PressPassUserMembershipsSerializer(serializers.ModelSerializer):
```

Comment:
It is an arbitrary decision, but I chose `Membership` to live in the `Organization` app, so it should be in `squarelet/organizations/serializers.py`

Reply:
Okay, no problem. I've moved it to the Organization app.